playbooks/roles/nginx-proxy/templates/nginx-proxy.j2

```diff
@@ -13,6 +13,24 @@ server {
 
     {{ NGINX_PROXY_EXTRA_CONFIG }}
 
+    set $expected_origin false;
+    if ($http_origin ~ {{ NGINX_PROXY_ORIGIN_REGEX }}) {
+    set $expected_origin true;
```

Comment:
This is not indented correctly so that the generated config is indented correctly. (following the pattern on the above lines)

Reply:
> This is not indented correctly so that the generated config is indented correctly. (

That's true for lines 21-27, but line not for line 18 above, because that `if` clause is part of nginx config and not jinja code.